semantic/tasks/watch.js

```diff
@@ -0,0 +1,231 @@
+/*******************************
+           Watch Task
+*******************************/
+
+var
+  gulp         = require('gulp-help')(require('gulp')),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5182945)